README.md

```diff
@@ -41,31 +41,7 @@ The governance of the project is described in more detail in the
 
 The Egeria team runs three types of calls and all are welcome.
 
-Calls are scheduled for one hour on Zoom Conference [https://zoom.us/j/523629111](https://zoom.us/j/523629111).
-
-[Egeria Calendar](https://lists.lfaidata.foundation/g/egeria-technical-discuss/calendar) holds the latest timings and information for these calls.
-
-* **Community call** 
-   * Every Wednesday.
-   * Demos, meet-ups and other activities going on in the Egeria community.
-   * *Egeria Community Call (every Wednesday)* in the
-   [Egeria Calendar](https://lists.lfaidata.foundation/g/egeria-technical-discuss/calendar).    
-   * Zoom Conference [https://zoom.us/j/523629111](https://zoom.us/j/523629111).
-   * [Egeria community meeting minutes](https://wiki.lfaidata.foundation/display/EG/Community+Meetings).
-  
-* **Technical Steering Committee (TSC)** 
-  * Every second Wednesday.
-  * Discussion on the management of the community.
-  * *Egeria TSC Meetings* in the
-  [Egeria Calendar](https://lists.lfaidata.foundation/g/egeria-technical-discuss/calendar).    
-  * [Egeria TSC home page](https://wiki.lfaidata.foundation/display/EG/Egeria+Technical+Steering+Committee+%28TSC%29+Home).
-
-* **Developers call** 
-   * Every Thursday. 
-   * Discussion on code development. 
-   * Not minuted. 
-   * *Egeria Developers Community Call (recorded)* in the
-     [Egeria Calendar](https://lists.lfaidata.foundation/g/egeria-technical-discuss/calendar).    
+Refer to [Community Guide](https://odpi.github.io/egeria-docs/guides/community/) for updated Call information.
```

Comment:
We should point into the new mkdocs here https://odpi.github.io/egeria-docs/guides/community/.